test/integration/script_lint_test.go

```diff
@@ -0,0 +1,100 @@
+package integration
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/nuodb/nuodb-helm-charts/test/testlib"
+)
+
+const shebangPattern = "[#][!][ \t]*/[^/]+/(sh|bash)"
+
+func TestLintScripts(t *testing.T) {
+	rootPaths := [...]string{
+		testlib.ADMIN_HELM_CHART_PATH,
+		testlib.DATABASE_HELM_CHART_PATH,
+		testlib.RESTORE_HELM_CHART_PATH,
+		testlib.THP_HELM_CHART_PATH,
+		testlib.YCSB_HELM_CHART_PATH,
+	}
+
+	shebang, err := regexp.Compile(shebangPattern)
+	if err != nil {
+		t.Error("Cannot compile patterh:", shebangPattern)
+	}
```

Comment:
require.NoError(...

Reply:
Sure.

And there is a typo in there as well!

Double value!! :)